### ISSUE TITLE
SliverGeometry is not valid: The "maxPaintExtent" is less than the "paintExtent".

### ISSUE THREAD

**Issue by felixjaehn** [Kavantix/sliver_tools#34]
This error occurs when using the MultiSliver.
The following video shows the error:

https://user-images.githubusercontent.com/31625592/121218421-01b76800-c883-11eb-93b8-5a32ac3bff0a.mp4

Code to reproduce this error:

```
class _MyHomePage extends StatelessWidget {
  @override
  Widget build(BuildContext context) {
    List<Widget> widgetList = List<Widget>.generate(
        5,
        (index) => MultiSliver(
              children: [
                SliverPersistentHeader(
                  pinned: true,
                  delegate: HeaderDelegate(),
                ),
                Container(height: 200, color: Colors.green),
                SliverPersistentHeader(
                  pinned: true,
                  delegate: HeaderDelegate(),
                ),
              ],
            ));
    return Scaffold(
        body: SafeArea(
      child: CustomScrollView(
        slivers: widgetList,
      ),
    ));
  }
}

class HeaderDelegate extends SliverPersistentHeaderDelegate {
  @override
  Widget build(
      BuildContext context, double shrinkOffset, bool overlapsContent) {
    return Container(height: 60, color: Colors.white);
  }

  @override
  double get maxExtent => 60;

  @override
  double get minExtent => 60;

  @override
  bool shouldRebuild(SliverPersistentHeaderDelegate oldDelegate) {
    return true;
  }
}
```

Flutter 2.2.1
Sliver Tools 0.2.4
Tested on Samsung Device (Android 10)

**Comment by Kavantix** [comment]
Thanks for filing this!
Could you check if #35 fixes your issue

**Comment by felixjaehn** [comment]
This indeed fixed the error. Thank you for being that quick!

**Comment by Kavantix** [comment]
Great! The fix is now also released as v0.2.5
